conf.yaml

```diff
@@ -626,33 +626,33 @@ contents:
                 repo:   cloud
                 path:   docs/shared
               -
```

Comment:
we're missing .NET, need to add the following block:
```
alternatives: { source_lang: console, alternative_lang: csharp }
repo:   clients-team
path:   examples/elastic-cloud/csharp
map_branches: *mapCloudSaasToClientsTeam
```

Reply:
Fixed, thank you for pointing this out!